spring-cloud-app-broker-docs/src/docs/asciidoc/deployment-platforms.adoc

```diff
@@ -1,4 +1,21 @@
 [[deployment-platforms]]
 == Deployment Platforms
 
+You can configure details of deployment platforms in App Broker configuration properties. These properties are under `spring.cloud.appbroker.deployer`. Currently, Spring Cloud App Broker supports only Cloud Foundry as a deployment platform.
 
+To configure a Cloud Foundry deployment platform, use properties under `spring.cloud.appbroker.deployer.cloudfoundry`, as in the following example:
+
+[subs=+quotes]
+```yml
+spring:
+  cloud:
+    appbroker:
+      deployer:
+        *cloudfoundry:*
+          *api-host: api.sys.example.com*
+          *api-port: 443*
+          *username: admin*
```

Comment:
An app has 2 options for authenticating to the configured CF: `username` and `password` *or* `client-id` and `client-secret` for OAuth2.